packages/backend/src/graphql/resolvers/peer.ts

```diff
@@ -122,6 +122,9 @@ export const peerToGraphql = (peer: Peer): SchemaPeer => ({
   id: peer.id,
   maxPacketAmount: peer.maxPacketAmount,
   http: peer.http,
+  incomingTokens: peer.incomingTokens?.map(
```

Comment:
I think `incomingTokens` would probably benefit from a separate resolver kind of like `liquidity` is done. Check `packages/backend/src/graphql/resolvers/index.ts`. This will require a new function in the `HttpTokenService` instead of doing `.withGraphFetched('incomingTokens')` in the getter in peer service.